defaults/main.yml

```
@@ -282,6 +285,22 @@ vault_tls_disable_client_certs: false
 # Seal variables
 # ---------------------------------------------------------------------------
 
+# transit auto unseal, see https://www.vaultproject.io/docs/configuration/seal/transit
+vault_transit: false
+vault_transit_backend: vault_seal_transit.j2
+vault_transit_config: "{{ vault_config_path }}/vault_transit.hcl"
+vault_transit_address: ''
+vault_transit_token: ''
+vault_transit_disable_renewal: false
+vault_transit_key_name: 'autounseal'
+vault_transit_mount_path: "transit/"
+#vault_transit_namespace: ''
+vault_transit_tls_ca_cert: "ca_cert.pem"
+vault_transit_tls_client_cert: "client_cert.pem"
+vault_transit_tls_client_key: "ca_cert.pem"
```

These variables should take the `_file` suffix to be consistent with other variables.

---

Also, the default values could be improved IMHO:
```suggestion
vault_transit_tls_ca_cert_file: "{{ vault_tls_ca_file }}"
vault_transit_tls_client_cert: "autounseal_client_cert.pem"
vault_transit_tls_client_key: "autounseal_client_key.pem"
```